formatter/fmt

```diff
@@ -75,12 +75,33 @@ function get_modified_html_files() {
   ) | grep -E ".+\.(html)$" || true
 }
 
-function get_modified_ts_json_md_and_html_files() {
+function get_modified_css_files() {
```

Comment:
(blocking) I don't think we need this. Since we'll be writing our css with Sass we shouldn't need to format css files. Any css files that are in the codebase are probably going to be from external sources or autogenerated from scss files. And we shouldn't format those.